src/libraries/System.Text.Json/gen/Resources/xlf/Strings.pl.xlf

```diff
@@ -82,6 +82,16 @@
         <target state="translated">Nieogólny element „JsonStringEnumConverter” wymaga dynamicznego kodu.</target>
         <note />
       </trans-unit>
+      <trans-unit id="JsonUnsupportedLanguageVersionMessageFormat">
```

Comment:
I'm looking at the translation above for JsonStringEnumConverterNotSupportedTitle and others... I think it's a unique skill to understand those translations